net/README.md

```diff
@@ -20,27 +30,156 @@ $ aws configure
 ```
 More information on AWS CLI configuration can be found [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html#cli-quick-configuration)
 
-### Metrics configuration (Optional)
-Ensure that `$(whoami)` is the name of an InfluxDB user account with enough
-access to create a new InfluxDB database.  Ask mvines@ for help if needed.
+## Metrics configuration (Optional)
+Metrics collection relies on 2 environment variables that are patched to the remote nodes:
+ * `RUST_LOG` to enable metrics reporting in principle
+ * `SOLANA_METRICS_CONFIG` to tell agave where to log the metrics
+
+### Preparation
+> [!NOTE]
+> Anza employees should follow the guide in notion to set up the influxDB account.
+
+ * Ensure that `${host}` is the host name of the InfluxDB you can access, for example `https://metrics.solana.com:9999`
```

Comment:
Just using the hard-coded `host=https://internal-metrics.solana.com:8086`
throughout this document will save people time using the wrong value here.
It is already hardcoded in the script:
https://github.com/anza-xyz/agave/blob/395db14a7/net/init-metrics.sh#L32

Reply:
switched to internal-metrics as you have suggested